lib/ansible/modules/network/fortimanager/fmgr_secprof_waf.py

```diff
@@ -0,0 +1,1633 @@
+    description:
+      - Maximum number of range segments in HTTP range line.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  constraint_exception_max_url_param:
+    description:
+      - Maximum number of parameters in URL.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  constraint_exception_method:
+    description:
+      - Enable/disable HTTP method check.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  constraint_exception_param_length:
+    description:
+      - Maximum length of parameter in URL, HTTP POST request or HTTP body.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  constraint_exception_pattern:
+    description:
+      - URL pattern.
+    required: false
+
+  constraint_exception_regex:
+    description:
+      - Enable/disable regular expression based pattern match.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  constraint_exception_url_param_length:
+    description:
+      - Maximum length of parameter in URL.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  constraint_exception_version:
+    description:
+      - Enable/disable HTTP version check.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  constraint_header_length_action:
+    description:
+      - Action.
+      - choice | allow | Allow.
+      - choice | block | Block.
+    required: false
+    choices: ["allow", "block"]
+
+  constraint_header_length_length:
+    description:
+      - Length of HTTP header in bytes (0 to 2147483647).
+    required: false
+
+  constraint_header_length_log:
+    description:
+      - Enable/disable logging.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  constraint_header_length_severity:
+    description:
+      - Severity.
+      - choice | low | Low severity.
+      - choice | medium | Medium severity.
+      - choice | high | High severity.
+    required: false
+    choices: ["low", "medium", "high"]
+
+  constraint_header_length_status:
+    description:
+      - Enable/disable the constraint.
+      - choice | disable | Disable setting.
+      - choice | enable | Enable setting.
+    required: false
+    choices: ["disable", "enable"]
+
+  constraint_hostname_action:
+    description:
+      - Action.
```

Comment:
Description is very generic, please update the `description` wherever needed. Also, need not to  mention choice in `description` field as you already mentioning the same in `choice` field